Source/buildbindinggo.go

```diff
@@ -586,6 +602,15 @@ func buildGoWrapper(component ComponentDefinition, w LanguageWriter) error {
 	w.Writeln("  \"fmt\"")
 	w.Writeln("  \"unsafe\"")
 	w.Writeln("  \"runtime\"")
+	
+	if len(component.ImportedComponentDefinitions) > 0 {
+		w.Writeln("  // Injected Components")
+		for _, subComponent := range(component.ImportedComponentDefinitions) {
+			subNameSpace := subComponent.NameSpace
+			w.Writeln("  \"../%s\"", subNameSpace)
```

Comment:
Relative imports does not work when using `modules`, which is the standard go dependency managment. [More info](https://github.com/golang/go/wiki/Modules#do-modules-work-with-relative-imports-like-import-subdir)

Reply:
I see, how would we integrate this then? Just leave away the ".."?

Reply:
Removing the `..` don't work, all imports that are not from the standard library needs to be prefixed with the name of the module, even if the package is in the same module. Therefor the name of the module should be known at generation time if you want to import another package.

Another approach would be to define an interface that is fulfilled by the injected component and use it instead of importing it, this way the generated code is decoupled from the location of the injected code. This is possible in Go but not in other languages because a struct doesn't need to specify that it is implementing an interface to be used as so.

Example using the calculator example:

```go
// NumberVariable is infered from Numbers.xml
type NumberVariable interface {
  GetValue() float64
  SetValue(float64)
}

type Calculator struct {
  ...
}
func (c *Calculator) GetEnlistedVariable(index uint32) NumberVariable {
  ...
}
```